tests/standards/OpenStudio_Standards_materials.json

```diff
@@ -0,0 +1,15579 @@
+{
+  "1 Coat Stucco": {
+    "name": "1 Coat Stucco",
+    "material_type": "StandardOpaqueMaterial",
+    "roughness": "MediumRough",
+    "thickness": 0.38,
+    "conductivity": 5.0004,
+    "resistance": 0.07599392049,
+    "density": 115.81,
+    "specific_heat": 0.2,
+    "material_standard": "CEC Title24-2013",
+    "material_standard_source": "CEC RJ",
+    "code_category": "Masonry Materials",
+    "code_identifier": "1 Coat Stucco",
+    "notes": "From CEC Title24 2013"
+  },
+  "1/2 in. Gypsum Board": {
+    "name": "1/2 in. Gypsum Board",
```

Comment:
Yes, the standards gem is full of materials like this one that take full advantage of EnergyPlus's lax rules around acceptable characters for names.  Right now, a lot of these characters are getting stripped out of the names when they get imported as Honeybee materials/constructions but I think that we should support these characters in the case of EnergyPlus materials, constructions, schedules, loads, and all other E+ objects (except geometry).  Stripping out acceptable E+ characters for every single object and storing the original name just does not seem like a very practical or elegant solution (I don't imagine that you store the original name for every Radiance object). 

You can see that this is a major reason why I think it is necessary to have [separate type-checking methods for Radiance and EnergyPlus names](https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/typing.py#L5-L53) since I don't think we should subject EnergyPlus materials to the same limitations as  Radiance materials (ie. no white spaces) and vice versa (ie. max of 100 characters).  I am planning for the next PR that I send to honeybee-core to have a better version of `valid_ep_string` that accepts all acceptable characters in EnergyPlus names.

For clarification, I think that stripping out all but a few types of characters and storing the original name is a good solution for honeybee-core objects like geometry because these objects get shared between EnergyPlus and Radiance simulations.  But, once we are working with objects that are specific to Radiance or EnergyPlus, the naming conventions of the engine should govern.

Reply:
Thank you for the detailed explanation. I like the plan. The only place that the name change will happen eventually is when a material / construction is translated to Radiance which in that case it should be fine anyways.

Reply:
Exactly.  You can see that I realized this as I was writing my response to you and I already pushed a change to my fork to ensure constructions get translated with valid Radiance names:
https://github.com/chriswmackey/honeybee-energy/commit/22f6e69b57933262eecb42a7dced90ca1756caa2

Reply:
You actually didn't need to do that. Each modifier takes care of changing the name to a valid radiance name. You can just pass the EP name string as is.

https://github.com/ladybug-tools/honeybee-radiance/blob/093eed49719dbef832b7f58b80c09e09ad4c9c78/honeybee_radiance/primitive/primitivebase.py#L242

Reply:
Ah, good point. Ok, I will take it out, then. There's no need to run the check twice.